webknossos/webknossos/annotation/annotation.py

```diff
@@ -124,6 +146,63 @@ def download(cls, annotation_path: str) -> "Annotation":
 
         return download_annotation(annotation_type, annotation_id)
 
+    def _skeleton_zip_path(self) -> Union[Openable, PathLike, str]:
```

Comment:
Why should this return a Union? Can't this simply return `_ZipPath` or `Openable` if we don't want to be too specific? I'm all in for Unions on arguments, but the return types should be rather specific IMO